### ISSUE TITLE
District boundary colors

### ISSUE THREAD

**Issue by adamboche** [JeffreyBLewis/WebVoteView#197]
From Howard:

> Use a different color to show a river as against a district boundary.  For example, in the 2nd House, the 2nd district of New York crosses the East River to include what is now the Bronx (and more) as well as Manhattan.

**Comment by aaronrudkin** [comment]
I think this is plainly implausible with the data we have currently. @JeffreyBLewis what do you think?

**Comment by JeffreyBLewis** [comment]
Yeah.  The problem is that some of the maps (probably recent ones from the census) are not clipped to land.   It would be a really good thing to address.  Let's leave it open.  Maybe I can find someone with good GIS skills address this.   Or maybe even me.

**Comment by aaronrudkin** [comment]
Jeff and I discussed this today, and his example was the same as Howard's.

Our rough plan is this:

- Get a map of major waterways of the US
- Overlay that map on top of the district map
- Only show the waterways if you zoom in past a certain point
- Make sure event polling bubbles down from the waterways to the districts

Separately:
- Get a state boundaries json and layer that on top of the district boundaries so that we can make state boundaries a little thicker.

There's no clear ETA on this but just wanted to highlight we're thinking about it.
